tcltests/test_helper.tcl

```diff
@@ -21,7 +21,6 @@ set dir [pwd]
 set ::all_tests []
 
 set test_dirs {
-    unit
```

Comment:
Hello Lucifer, please include unit folder again, thanks

Reply:
My bad, missed it. Thanks will fix